### PR TITLE
NMS-14051: Update existing documentation related to provisiond xml file

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14051

### CODE REVIEW

File: docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ config:edit org.opennms.sentinel.controller
 config:property-set location SENTINEL<1>
```
Comment (mmahacek) [inline docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc:26]: This file is missing a top level heading and is not part of the navigation.

File: docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc [hunk 2]
```diff
@@ -25,6 +25,7 @@ ssh -p 8301 admin@localhost
 config:edit org.opennms.sentinel.controller
```
Comment (mmahacek) [inline docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc:25]: This file is missing a top level heading and is not part of the navigation.

File: docs/modules/reference/pages/provisioning/handlers/vmware.adoc [hunk 3]
```diff
@@ -184,7 +181,7 @@ Enabling the https://opennms.discourse.group/t/storing-data-with-foreign-sources
 
 == Provision a subset of VMware entities
 
-If you want to provision an arbitrary selection of VMware entities, you can specify a key-value pair to match objects that have a specific attribute in vCenter.
+To provision an arbitrary selection of VMware entities, you can specify a key-value pair to match objects that have a specific attribute in vCenter.
```
Comment (mmahacek) [inline docs/modules/reference/pages/provisioning/handlers/vmware.adoc:184]: Are these examples still valid for the v30 importer where there is no XML?

**Review by mmahacek — COMMENTED** [review]
Just a few questions.  I haven't seen the CM stuff in person yet.

File: docs/modules/reference/pages/provisioning/handlers/vmware.adoc (hunk 3 above)
Comment (Bonrob2) [reply docs/modules/reference/pages/provisioning/handlers/vmware.adoc:184]: Yes, there is an Advanced Options section that lists all of these same parameters. Will need Jeff to verify that the default values are still valid.

File: docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc (hunk 1 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc:26]: It is content in a tab (like we use for the different installation OSs), so doesn't get a top level heading and is not part of the nav.

File: docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc (hunk 2 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/flows/sentinel/message-broker/kafka.adoc:25]: It is content in a tab (like we use for the different installation OSs), so doesn't get a top level heading and is not part of the nav.

File: docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc (hunk 1 above)
Comment (mmahacek) [reply docs/modules/operation/pages/flows/sentinel/message-broker/activemq.adoc:26]: ah, missed that.

**Review by mmahacek — APPROVED** [review]
LGTM, assuming the VMware advanced provisioning instructions are correct.

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 4]
```diff
@@ -66,7 +66,7 @@ NOTE: If any syntactical or XML structural problems occur in the requisition, th
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/introduction.adoc:66]: Line 47: Should this still say "XML definition"?

Comment (Bonrob2) [reply docs/modules/operation/pages/provisioning/introduction.adoc:66]: No. Updated.

**Review by mmahacek — DISMISSED** [review]
LGTM

**Comment by Bonrob2** [comment]
@jeffgdotorg or @indigo423 , this is ready for review with the changes in the UI we discussed previously.

**Comment by Bonrob2** [comment]
@jeffgdotorg are you able to review?

**Review by mmahacek — DISMISSED** [review]
I don't think I've got anything that would prevent merging at this time.  Looks good.

**Review by jeffgdotorg — CHANGES_REQUESTED** [review]
Need some changes described in comments.

File: core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/requisitions.html [hunk 5]
```diff
@@ -38,7 +38,7 @@ <h4>Requisitions ({{ requisitionsData.requisitions.length }})</h4>
           <span class="fa fa-retweet"></span> Reset Default
         </button>
         <button type="button" class="btn btn-secondary" ng-click="add()" id="add-requisition" ng-disabled="timingStatus.isRunning">
-          <span class="fa fa-plus"></span> Add Requisition
+          <span class="fa fa-plus"></span> Add Requisition Definition
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/requisitions.html]: This has been updated.

File: core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/requisitions.html [hunk 6]
```diff
@@ -62,8 +62,8 @@ <h1>Loading Requisitions</h1>
     <!-- No Requisitions -->
     <div class="jumbotron" ng-show="loaded && requisitionsData.requisitions.length == 0">
       <div class="container">
-        <h1>There are no requisitions</h1>
-        <p>Please click on the 'Add Requisition' button, to start adding requisitions.</p>
+        <h1>There are no requisition definitions</h1>
+        <p>Click the 'Add Requisition Definition' button, to add some.</p>
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/views/requisitions.html]: Also updated to External Requisitions. Not sure why this is showing as is.

**Review by jeffgdotorg — APPROVED** [review]
Looks good. Thanks Bonnie!

**Review by maximbrener — APPROVED** [review]

**Pull request merged**